project.clj

```diff
@@ -1,6 +1,6 @@
-(defproject clojure-opennlp "0.5.1-SNAPSHOT"
+(defproject org.clojars.dmitrykosykh/clojure-opennlp "0.5.1-SNAPSHOT"
```

Comment:
I think these changes are accidental and should not be included in this PR

Reply:
I can't understand why the second commit was added to PR